glancesync/getnid.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
+# -- encoding: utf-8 --
+#
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-Core project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+"""Get the NID of the different GE(r)i.
+
+Usage:
+  getnid [--wikitext]
+  getnid --type (i2nd | cloud | userinterface | sec | iot | data | apps)
+  getnid -h | --help
+  getnid -v | --version
+
+Options:
+  -h --help     Show this screen.
+  -v --version  Show version.
+  --wikitext    Generate content in Wiki Markup format.
+  --type        Show details about the specific chapter
+
+"""
+
+author = 'fla'
+
+from docopt import docopt
+import requests
+import pprint
+import re
+
+__version__ = '1.0.0'
+
+
+class NID(object):
+    """Class to obtain the different nids values registered in the FIWARE Catalogue.
+    """
+    def __init__(self):
+        self.BASEURL = 'http://catalogue.fiware.org/chapter/'
```

Comment:
What do you think of having a `getnid.cfg` with these configurations (BASEURL, CHAPTER, TYPE)?